ete4/gtdb_taxonomy/gtdbquery.py

```diff
@@ -146,13 +146,29 @@ def _translate_merged(self, all_taxids):
 
     #     return taxid, spname, norm_score
 
-    def get_rank(self, taxids):
+    def _dirty_id_suffix(self, taxid):
+        pass
+
```

Comment:
I don't see this function used. Why is it defined?

Reply:
This is a function I want to do to handle GTDB accesssion,
because I realized sometime they use ids in both way, for example:
GCA_000003645.1	 GB_GCA_000003645.1
GCF_900113245.1   RS_GCF_900113245.1
they are equivalent so I want to have a function to handle the fuzzy. But I will keep it in my dev repo

Reply:
Great, thanks!